src/main/resources/fabric.mod.json

```diff
@@ -1,36 +1,40 @@
 {
   "schemaVersion": 1,
   "id": "carpet-autocraftingtable",
-  "version": "1.4.17",
+  "version": "${version}",
 
-  "name": "Auto-Crafting Table mod for carpet-mod",
-  "description": "Auto-Crafting Table mod for carpet-mod originally by Skyrising",
+  "name": "Carpet-AutoCraftingTable",
```

Comment:
I still don't get why changing another one's mod name.

Also, autoversioning would break horribly here, since it needs to be adapted in build.gradle too (Carpet mod and related change the $version variable to be mcver-ver, I had to touch that in main Carpet).

Reply:
> I still don't get why changing another one's mod name.

Because conciseness is nice for mod names, especially to fit in with other mods, and it is nice to have parity with the mod's .jar file name in your file manager.

Reply:
I have reverted the autoversioning change, feel free to create another PR for that.